vllm/v1/core/kv_cache_manager.py

```diff
@@ -9,6 +10,141 @@
 logger = init_logger(__name__)
 
 
+@dataclass
+class KVCacheBlock:
```

Comment:
Maybe add a last access time field for LRU?

Reply:
Oh after a second look I think LRU is implicitly achieved by the order in the queue.